src/package.json

```diff
@@ -19,7 +19,8 @@
     "web3": "^0.20.7"
   },
   "devDependencies": {
+    "babel-core": "^6.26.3",
```

Comment:
is babel-core required?

Reply:
it's a dev dependency and parcel depends on it